Data/Primitive/MVar.hs

```diff
@@ -0,0 +1,133 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE MagicHash #-}
+{-# LANGUAGE UnboxedTuples #-}
+
+-- |
+-- Module      : Data.Primitive.ByteArray
+-- Copyright   : (c) Roman Leshchinskiy 2009-2012
```

Comment:
fix ther module name and you can drop the copyright and maintainer fields

set license for the module to BSD2 or something, style is ambiguous :) 

(we should add an authors file though?)